### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by christina-moore** [SocialFinanceDigitalLabs/cs-demand-model-regional#95]
The application needs to meet AA accessibility standards. We should investigate the appropriate tools we want to implement. For now, we conducted an assessment with the EqualWeb chrome extension and the following issues were raised, organised by view:

**Overall**
- Contrast on navbar headers (4.5:1)
- <title> within <head> block for each page
**Predict**
- two submit buttons with same ID - leverage django form prefixes
- 3 of 4 blue boxed filter fields are not accessible to screen readers: "This searchinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined." ([Further reading](https://www.w3.org/WAI/standards-guidelines/act/rules/e086e5/) on this one)
**Scenarios**
- Anchor element found with a valid href attribute, but no link content has been supplied. Link with a href attribute, but without link text. This means it does not have an accessibility API "name".
**Historic Data**
- search elements need valid names (same as predict)
- anchor elements with no link and no name/id (same as scenarios)
**Adjusted**
- search elements need valid names
- anchor elements with no link and no name/id
- two submit buttons with same ID (same as predict)
- two elements with id="efficiency-hub-table"
- "The relationship between td elements and their associated th elements is not defined. As this table has multiple levels of th elements, you must use the headers attribute on td elements. Table where not all TH elements contain IDs"."
**Costs**
- td elements (same as adjusted)
- anchor elements with no links
- two elements with id="efficiency-hub-table" and id="symbol"
